cryostat/templates/secret.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if empty .Values.core.databaseSecretName -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cryostat-jmx-credentials-db
+  namespace: {{ .Release.Namespace }}
```

Comment:
I am quite unsure to set this namespace as other resources do not. Any guide on this?

Reply:
There is a long discussion here on this topic: https://github.com/helm/helm/issues/5465. The maintainers seem to suggest omitting the namespace.

Reply:
Right I will remove it then :D